src/components/Checkbox/Checkbox.js

```diff
@@ -145,18 +139,10 @@ const Checkbox = ({
 };
 
 Checkbox.propTypes = {
-  /**
-   * Controles/Toggles the checked state.
-   */
-  onChange: PropTypes.func.isRequired,
```

Comment:
I'd keep the prop type here, but remove the `isRequired` and add a default prop with value `null`. WDYT?